clients/drcachesim/tracer/tracer.cpp

```diff
@@ -245,6 +247,7 @@ event_bb_setup(void *drbbdup_ctx, void *drcontext, void *tag, instrlist_t *bb,
     DR_ASSERT(enable_dups != NULL && enable_dynamic_handling != NULL);
     if (bbdup_duplication_enabled()) {
         *enable_dups = true;
+        /* make sure to update opts.non_default_case_limit if adding an encoding here */
```

Comment:
style nit: Prefer capitalized and punctuated full sentences in comments.

Reply:
Done.